packages/stryker-mutator-specification/src/ForStatementMutatorSpec.ts

```diff
@@ -0,0 +1,22 @@
+import * as os from 'os';
+import { expect } from 'chai';
+import ExpectMutation from './ExpectMutation';
+
+export default function ForStatementMutatorSpec(name: string, expectMutation: ExpectMutation) {
+
+  describe('ForStatementMutator', () => {
+
+    it('should have name "ForStatement"', () => {
+      expect(name).eq('ForStatement');
+    });
+
+    it('should mutate the condition of a for statement', () => {
+      expectMutation('for(let i=0;i<10; i++) { console.log(); }', 'for(let i=0;false; i++) { console.log(); }');
+    });
+
+    it('should mutate the condition of a for statement without a condition', () => {
+      expectMutation('for(let i=0;; i++) { console.log(); }', `for (let i = 0; false; i++) {${os.EOL}    console.log();${os.EOL}}`);
```

Comment:
We are formatting our code here. Is that intentional?  We don't format the previous for statement. How will we make sure that all mutators return this exact formatting change?

Reply:
Agreed, let's make the code here readable and change the expect implementation for typescript.